conf/report/report.css

```diff
@@ -15,6 +15,36 @@ body {
   height: 150px;
 }
 
+.report_table_info {
+  position: sticky;
+  left: 0;
+  z-index: 2;
+  background-color: white;
+}
+
+.report_table_tag {
+  position: sticky;
+  left: 31em;
```

Comment:
`calc()` should be used here to sum first column's width and padding. Otherwise there is slight misalignment:
![Screenshot_20210514_153113](https://user-images.githubusercontent.com/4888536/118279002-e8610d00-b4ca-11eb-810f-2971f5576f03.png)